packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain/useMaxAmount.ts

```diff
@@ -36,7 +39,7 @@ export function useMaxAmount() {
     if (nativeCurrency.isCustom && isDepositMode) {
       return utils.formatUnits(
         nativeCurrencySourceBalance,
-        nativeCurrency.decimals
+        nativeCurrencyDecimalsOnSourceChain
```

Comment:
need to make the same change on L49, otherwise the max amount for withdrawals will be messed up

<img width="601" alt="image" src="https://github.com/user-attachments/assets/ddf9f0cb-e1e1-4239-afbc-7cc556d51475">

Reply:
done